addons/mail/tests/common.py

```diff
@@ -74,6 +74,9 @@ def _mail_mail_unlink(model, *args, **kwargs):
             self.send_email_mocked = send_email_mocked
             yield
 
+        if mail_unlink_sent:
```

Comment:
Maybe add a comment to say this is done to be backward compatible with tests before the update of this patch ?